packages/devextreme/js/ui/rating_types.d.ts

```diff
@@ -0,0 +1,10 @@
+export {
+  RatingSelectionMode,
+  ContentReadyEvent,
+  DisposingEvent,
+  InitializedEvent,
+  OptionChangedEvent,
+  ValueChangedEvent,
+  dxRatingOptions,
```

Comment:
Should we export both dxRatingOptions & Properties? I this that no. Moreover, dxRatingOptions is deprecated

Reply:
This file is automaticaly generated. also I see that other components (chat for examplae) also includes this

Reply:
I am not sure if dxRatingOptions should be public and whether we should declare and deprecate it. Kindly ask the API reviewers about this, as in DateRangeBox, we only add the public type Properties.
https://github.com/EugeniyKiyashko/DevExtreme/blob/24_2_security_issue/packages/devextreme/js/ui/date_range_box.d.ts#L170-L171